modules/ROOT/pages/activity-tracking.adoc

```diff
@@ -32,32 +32,33 @@ Once you have created a message flow, you can begin to track its activity.
 
 . Select *Activity* from the left navigation.
 
-. Locate a transmission you are interested in viewing. Use the search box to find the transmission based on:
+. Locate a transmission you are interested in viewing. Use the search box to find the transmission based on the:
 +
-** the trading partner name
-** the message type
-** the filename
-** the control number
-** the endpoint
-** the date the file was received
+* trading partner name
+* message type
+* filename
+* control number
+* endpoint
+* date the file was received
 
 . Select the transmission and review the summary information. The summary information contains:
 +
-** The status of the transmission indicating either a success (Green check mark), or failure (red circle with an exclamation point).
+* The status of the transmission indicating either a success (Green check mark), or failure (red circle with an exclamation point).
 
-** The name of the file that was received.
+* The name of the file that was received.
 
-** The control number associated with the file (if available).
+* The control number associated with the file (if available).
 
-** The receiving endpoint at which the file was received.
+* The receiving endpoint at which the file was received.
 
-** The date and time when the file was received.
+* The date and time when the file was received.
 
-** The types of messages received (Depending on the message flows you configured).
+* The types of messages received (Depending on the message flows you configured).
 
-** The number of messages received in the file.
+* The number of messages received in the file.
 +
-. Scan the Transmission Flow sections for an exclamation point (!) in any of the rectangle boxes next to the displayed flow stages. This indicates there is an issue with the transmission.
+. Scan the Transmission Flow sections for an exclamation point (!) in any of the rectangle boxes next to the displayed flow stages.
+This indicates there is an issue with the transmission.
 +
 
 The Transmission Flow section shows the steps the file was processed in. For example, if:
```

Comment:
The text intro and this list do not make sense. And I'm not sure that it is even necessary, because the previous step implies that stages are shown, with indications if any was unsuccessful.
If anything, I suggest that we briefly extend the end of that step, something like this:
"...next to the displayed flow stages (for example, message reception, routing, acknowledgment, and so on)."

Reply:
Next release.